presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientConfig.java

```diff
@@ -138,4 +142,52 @@ public KuduClientConfig setSchemaEmulationEnabled(boolean enabled)
         this.schemaEmulationEnabled = enabled;
         return this;
     }
+
+    public boolean isKerberosAuthEnabled()
+    {
+        return kerberosAuthEnabled;
+    }
+
+    @Config("kudu.kerberos-auth.enabled")
+    public KuduClientConfig setKerberosAuthEnabled(boolean enabled)
+    {
+        this.kerberosAuthEnabled = enabled;
+        return this;
+    }
+
+    public String getKerberosPrincipal()
+    {
+        return kerberosPrincipal;
+    }
+
+    @Config("kudu.kerberos-auth.principal")
```

Comment:
Kerberos related properties should not be allowed to be configures if `kudu.authentication.type=NONE`. Please see usages how it typically achieved by look at usages of `io.airlift.configuration.ConditionalModule#installModuleIf`.